INSTALL.rst

```diff
@@ -2,6 +2,41 @@
 Installation
 ============
 
+System prerequisites
+=====================
+
+This guide expects you to have installed is your system the following tools:
```

Comment:
that's generic, you'll need those tools in any system where you plan to run the dev environment, were they installed by default or not. My (CentOS) system specifics are below.